kolibri/core/content/fixtures/content_test.json

```diff
@@ -387,6 +452,51 @@
     "pk": "8ad3fffedf144cba9492e16daec1e39a",
     "model": "content.localfile"
   },
+  {
+    "fields": {
+      "available": true,
+      "extension": "png",
+      "file_size": null
```

Comment:
My only concern is with setting the file size of these to `null`. Our current fixture is terrible and already has some completely unrealistic data in it, so my thought is not to make it any more unrealistic by having files with null size.

I assume you maybe tried with a non-null file size and lots of tests broke?

Reply:
Ah - in that case, we can leave as is, it's been a long time since I manually edited this fixture!

Reply:
I changed them to 1 and it just changed the test expectations.